staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1beta1/conversion.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package v1beta1
 
 import (
-	conversion "k8s.io/apimachinery/pkg/conversion"
-	clientauthentication "k8s.io/client-go/pkg/apis/clientauthentication"
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/client-go/pkg/apis/clientauthentication"
 )
 
 func Convert_clientauthentication_ExecCredentialSpec_To_v1beta1_ExecCredentialSpec(in *clientauthentication.ExecCredentialSpec, out *ExecCredentialSpec, s conversion.Scope) error {
```

Comment:
add a comment indicating this intentionally excludes response and interactive, which were only supported in v1alpha1

Reply:
https://github.com/kubernetes/kubernetes/pull/95489/commits/bd4af0df52b7da2a838795ccf9a8760a0316da48